conf/prod.routes

```diff
@@ -1,5 +1,3 @@
 # Add all the application routes to the app.routes file
 ->         /                                      app.Routes
 ->         /                                      health.Routes
-
-GET        /admin/metrics             com.kenshoo.play.metrics.MetricsController.metrics
```

Comment:
I was told that this isn't being used (since we're not deploying to production) and can be removed.

Reply:
Is this not being consumed by anything else during acceptance tests? Please run all acceptance tests before removing this

Reply:
> Please run all acceptance tests before removing this

I don't know where *all* the ATs are.

> is it used in staging for performance tests

How do I find out?

Reply:
> Please run all acceptance tests before removing this

Thanks for the help! I ran all of them and they passed after some more tweaks with the versions.

> is it used in staging for performance tests

We agreed to deploy it (after reading the docs) and verify that Grafana is still working.